native/NativeSampleApps/SmartSyncExplorer/src/com/salesforce/samples/smartsyncexplorer/ui/MainActivity.java

```diff
@@ -132,6 +132,15 @@ protected void onCreate(Bundle savedInstanceState) {
 	}
 
 	@Override
+	protected void logoutCompleteActions() {
+		super.logoutCompleteActions();
```

Comment:
Maybe we don't really need to pass the activity to logout and have the SalesforceSDKManager finish activities.